index.php

```diff
@@ -37,7 +40,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
 
 <head>
-	<title>Plugins for Geany</title>
+	<title>Plugins for Geany [<?php print PAGETITLE ?>]</title>
```

Comment:
here you just throw it out without escaping it, watch htmlspecialchars and so on

Reply:
ah no wait, github just skipped braces, pagetitle is only set when a file with this name exists so it should be fine